libkbfs/keybase_service_util.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"github.com/keybase/client/go/logger"
+	"golang.org/x/net/context"
+)
+
+// serviceLoggedIn should be called when a new user logs in. It
+// shouldn't be called again until after serviceLoggedOut is called.
+func serviceLoggedIn(ctx context.Context, log logger.Logger, name string,
+	hasSession hasSession, config Config,
+	bws TLFJournalBackgroundWorkStatus) {
+	const sessionID = 0
+	session, err := hasSession.CurrentSession(ctx, sessionID)
```

Comment:
Since this method takes a full config, including a fully functional KeybaseService in all the test callers, it doesn't seem very useful to make a separate interface.  Why not just have it use `config.KeybaseService()` and not complicate things with an extra interface?